app/assets/stylesheets/pages/_store_catalog.scss

```diff
@@ -1,91 +1,184 @@
 
 .info-container {
-  margin-bottom: 2%;
+  display: flex;
+  flex-direction: column;
+  height: 60px;
+  width: 80%;
+  margin-bottom: 55px;
 }
 
-.store-name {
-  font-size: 35px;
+.catalog-left {
+  display: flex;
+  margin-right: 40px;
 }
 
 .catalog-container {
-  width: 80%;
-  margin-left: 10%;
-  margin-right: 10%;
-  padding: 24px;
-  height: 500px;
+  width: 60%;
+  padding-left: 24px;
   display: table;
-  border-radius: 20px;
+  justify-content: center;
+}
+
+.catalog-body {
+  margin-left: 5%;
+}
+
+.product-container {
+  overflow: auto;
+  display: block;
+  height: 390px;
 }
 
 .catalog {
+  height: 392px;
+  width: 1100px;
   border-collapse: collapse;
   overflow: auto;
-  width: 100%;
 }
 
-.catalog-top {
-  display: flex;
-  justify-content: space-between;
+.store-balance {
+  margin-bottom: 10px;
+  margin-top: 20px;
+  height: 24px;
+  color: #353434;
+  font-family: 'Work Sans';
+  font-size: 16px;
+  font-weight: 500;
+  line-height: 24px;
+  margin-left: 25px;
 }
 
 .catalog-title {
-  font-size: 1.5em;
-  text-align: left;
-  margin-bottom: 2%;
+  height: 60px;
+  width: 196px;
+  color: #353434;
+  font-family: 'Work Sans', sans-serif;
+  font-size: 37px;
+  letter-spacing: -0.4px;
+  line-height: 60px;
+  margin-left: 25px;
+  margin-right: -20px;
 }
 
 .catalog-header-container {
   width: 100%;
+  height: 33px;
 }
 
 .catalog-header {
-  margin-bottom: 1%;
   display: flex;
   flex-direction: row;
-  flex-wrap: wrap;
-  width: 100%;
-
-  border-top: 1px solid rgb(189, 186, 186);
-  border-bottom: 1px solid rgb(189, 186, 186);
+  margin-bottom: -11px;
+  height: 40px;
 
   &__label {
     padding-top: 1%;
     padding-bottom: 1%;
     display: flex;
     flex-direction: column;
-    flex-basis: 100%;
-    flex: 1;
-
+    height: 16px;
+    color: #949494;
+    font-family: 'Work Sans';
     font-size: 14px;
-    font-weight: bold;
-    text-align: left;
+    font-weight: 500;
+    line-height: 16px;
   }
 }
 
-.new-product-btn {
-  font-size: 15px;
+.row-label__big {
+  flex-basis: 50%;
+  text-align: left;
 }
 
-.catalog-body {
-  margin-top: 1%;
+.row-label__medium {
+  flex-basis: 17%;
+  text-align: center;
+}
+
+.row-label__small {
+  flex-basis: 8.5%;
+  text-align: center;
+}
+
+
+
+.new-product-btn {
+  height: 40px;
+  width: 215px;
+  border-radius: 30.5px;
+  background-color: #35c6ad;
+  border: 0;
+  line-height: 0;
+  color: $white;
+  text-transform: uppercase;
+  font-family: 'Work Sans';
+  font-size: 14px;
+  font-weight: 600;
+  letter-spacing: 0.4px;
+  text-align: center;
+  margin-left: 40px;
+  margin-top: 10px;
+  margin-bottom: 10px;
 }
 
 .product {
   display: flex;
   flex-direction: row;
-  flex-wrap: wrap;
   width: 100%;
-
-  &:hover {
-    background-color: rgb(149, 202, 252);
-  }
+  line-height: 25px;
+  height: 56px;
+  border-bottom: 1px solid rgba(148,148,148,0.16);
```

Comment:
Expected single space after "," in a single-line function   function-comma-space-after<br>Unexpected leading zero                                     number-leading-zero